### PR TITLE
included travis ci file and makefile

### PR DESCRIPTION
**Pull request by puneethreddy20**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Go parameters
+
+GOCMD=go
```
Comment (cviecco) [inline Makefile]: simplify this. No need for variable names here (you are actually typing more and requiring more state to keep this)

File: api.go [hunk 2]
```diff
@@ -13,9 +13,13 @@ import (
 //Display all groups in Target LDAP --required
 func (state *RuntimeState) GetallgroupsHandler(w http.ResponseWriter, r *http.Request) {
 	var AllGroupsTargetLdap GetGroups
-
+	var err error
+	state.targetLdap,err=state.GetTargetLDAPConnection()
```
Comment (cviecco) [inline api.go]: All around here you will fail on concurrency. Change this to be local variables

File: templates/index.html [hunk 3]
```diff
@@ -183,7 +183,7 @@
 
     $(document).ready(function() {
         $('#pending_actions').DataTable( {
-            data: {{.Pending_actions}},
+            data: {{.PendingActions}},
```
Comment (cviecco) [inline templates/index.html]: was this working before? (my jquery-foo is weak)

Comment (puneethreddy20) [reply templates/index.html]: Yes. It was working before. Last time, when i changed the naming of some variables in Backend. I forgot to update in the html.

**Pull request merged**